checkpoint_quantum_firewall/assets/logs/checkpoint-quantum-firewall.yaml

```diff
@@ -0,0 +1,705 @@
+    name: Operation
+    path: checkpoint_operation
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: OS Name
+    path: os_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Protection Name
+    path: protection_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Protection Type
+    path: protection_type
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Protocol
+    path: checkpoint_firewall_protocol
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Reject Category
+    path: reject_category
+    source: log
+    type: string
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Server Inbound Packets
+    path: server_inbound_packets
+    source: log
+    type: integer
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Server Outbound Packets
+    path: server_outbound_packets
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Severity
+    path: checkpoint_firewall_severity
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: SmartDefense Profile
+    path: smartdefense_profile
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source
+    path: src
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source Machine Name
+    path: src_machine_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Status
+    path: checkpoint_firewall_status
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Subject
+    path: checkpoint_firewall_subject
+    source: log
+    type: string
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Total Bytes
+    path: checkpoint_firewall_bytes
+    source: log
+    type: integer
+    unit:
+      family: bytes
+      name: byte
+pipeline:
+  type: pipeline
+  name: Checkpoint Quantum Firewall
+  enabled: true
+  filter:
+    query: "source:checkpoint-quantum-firewall"
+  processors:
+    - type: service-remapper
+      name: Define `product` as the official service of the log
+      enabled: true
+      meta:
+        tags: []
+      sources:
+        - product
+    - type: date-remapper
+      name: Define `time` as the official date of the log
+      enabled: true
+      meta:
+        tags: []
+      sources:
+        - time
+    - name: Lookup for `severity` to `checkpoint_firewall_severity` field
```

Comment:
For this, we decided not to map severity with status because the values from checkpoint were Informational, Low, Medium, High, Critical which didn't match 1:1 with status field of datadog(Emergency, Alert, Critical, Warning, Notice, Info, Debug)

Reply:
Even if it didn't match exactly, they seem that they could be mapped bringing a lot of value to the customer in terms of UI.

Reply:
Which status the `severity` attribute will be mapped to can be customised by the integration pipeline based on a [category processor](https://docs.datadoghq.com/logs/log_configuration/processors/?tab=ui#category-processor). I strongly advise to use a category processor to remap the value of the severity attribute to the appropriate values of the status attribute.

Reply:
Unless I'm understanding things wrong and what you mean is that the `severity` attribute does not represent the status of the log because the real `status` of the log is represented by other attribute or is not represented at all in the log.